bootloader/Kconfig

```diff
@@ -0,0 +1,27 @@
+mainmenu "SC-OBC Module A1 Bootloader"
+
+menu "Application"
```

Comment:
Should this be "Application"? How is this presented in the menuconfig?

Reply:
Ah.. That's right.
Will change to `Bootloader`.